### DISCUSSION TITLE
How can i use if-else statements

### DISCUSSION THREAD

**Discussion by WithSJ** [CsqLang/Csq3#14] [Q&A] [answered]
**How can I use if-else statements**
I didn't check that but I think you did not write code for If-Else statements

**Comment by AniketKumar2500** [comment]
Ok so like this use if else:
if condition so
...
ends
else if condition so
...
ends
else starts
...
ends

**Reply by WithSJ** [reply]
have you used visual basic language because there we use " if condition then " and it's similar to that :)

and 
**the main point is >** 

if we have "start" and "ends" why do you use "so" this break our language rules we discussed that "start" and "ends" we will use as code block but you use "ends" without "start" I think it's not good

**Comment by AniketKumar2500** [comment]
it's because i want to seperate control statements like for,if,else,else if with block of code such as classes, functions.

**Comment by AniketKumar2500** [comment, marked as answer]
but yes i am also thinking to replace so for condition statements so.

**Comment by WithSJ** [comment]
In python when we write IF-ELSE, FOR, CLASS, and FUNCTIONS all have the same syntax 

Statement  colen
4 spaces indentation than its Code block start

Statement:
----Statement

**One Rule implemented on all statements that's the reason Python is easy and simple for our brain**



**IF Conditions**
```
if condition:
    pass
```
**For LOOP**
```
for i in range(5):
    pass
```
**Functions**
```
def ABC():
    pass
```
**Class**
```
class MyClass:
    pass
```

**Reply by AniketKumar2500** [reply]
Yeah right so lets make them look similar but i am still puzzled that shall i use short form the keyword starts to remove it's verbose natue?

**Reply by WithSJ** [reply]
I think the main key point of language should be simple and easy to remember 
right now it's already short we do not need too many short keywords
